CONTRIBUTING.md

```diff
@@ -8,34 +8,39 @@ If you have questions about this document or anything not covered here? Come cha
 
 - All code and doc submissions are done through pull requests against the `devel` branch.
 - Take care to make sure no merge commits are in the submission, and use `git rebase` vs `git merge` for this reason.
-- We ask all of our community members and contributors to adhere to the [Ansible code of conduct](http://docs.ansible.com/ansible/latest/community/code_of_conduct.html). If you have questions, or need assistance, please reach out to our community team at [codeofconduct@ansible.com](mailto:codeofconduct@ansible.com)   
+- We ask all of our community members and contributors to adhere to the [Ansible code of conduct](http://docs.ansible.com/ansible/latest/community/code_of_conduct.html). If you have questions, or need assistance, please reach out to our community team at [codeofconduct@ansible.com](mailto:codeofconduct@ansible.com)
 
 ## Setting up your development environment
 
-In this example we are using [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/), but any virtual environment will do.
+Install `tox` if it is not already installed on your system. This can be done with `pip` or your system package manager.
 
 ```bash
-(host)$ pip install virtualenvwrapper
-(host)$ mkvirtualenv ansible-runner
-(host)$ pip install -e .
+pip install tox -c test/constraints.txt
 ```
 
-When done making changes, run:
+Setup a test virtual environment and activate it:
 
 ```
-(host)$ deactivate
+tox -e dev
+source .tox/dev/bin/activate
 ```
 
-To reactivate the virtual environment:
+When done making changes, run:
 
 ```
-(host)$ workon ansible-runner
+deactivate
 ```
-## Linting and Unit Tests
 
-`tox` is used to run linters (`flake8` and `yamllint`) and tests.
+To reactivate the virtual environment:
 
 ```
-(host)$ pip install tox
-(host)$ tox
+source .tox/dev/bin/activate
 ```
+
+## Testing
+
+`tox` is used to run tests. To run the default set of tests, just run `tox`.
+
+To list all available test targets, run `tox -a`.
```

Comment:
It's better to be verbose, it'll show descriptions for every env, then:
```suggestion
To list all available test targets, run `tox -av`.
```

Example output (from a navigator checkout):
```console
$ tox -av                         
using tox.ini: ~/wk/src/github/ansible/ansible-navigator--bugfixes--503-diff-local-ansible-interpreter/tox.ini (pid 781911)
using tox-3.23.0 from ~/wk/.pyenv/versions/3.9.0/lib/python3.9/site-packages/tox/__init__.py (pid 781911)
default environments:
linters    -> Enforce quality standards under ~/wk/.pyenv/versions/3.9.0/bin/python3.9 (~/wk/src/github/ansible/ansible-navigator--bugfixes--503-diff-local-ansible-interpreter/.tox/linters/bin/python)
type       -> Verify static typing with MyPy under ~/wk/.pyenv/versions/3.9.0/bin/python3.9 (~/wk/src/github/ansible/ansible-navigator--bugfixes--503-diff-local-ansible-interpreter/.tox/type/bin/python)
py38       -> Run pytest under python3.8 (~/wk/src/github/ansible/ansible-navigator--bugfixes--503-diff-local-ansible-interpreter/.tox/py38/bin/python)
report     -> Produce coverage report
clean      -> Erase coverage data

additional environments:
smoke      -> Run smoke tests under ~/wk/.pyenv/versions/3.9.0/bin/python3.9 (~/wk/src/github/ansible/ansible-navigator--bugfixes--503-diff-local-ansible-interpreter/.tox/smoke/bin/python)
build-docs -> Build The Docs
```